Tests/FoundationEssentialsTests/AttributedString/AttributedStringCOWTests.swift

```diff
@@ -15,9 +15,9 @@ import TestSupport
 #endif
 
 #if FOUNDATION_FRAMEWORK
-@testable import Foundation
```

Comment:
Since `TestSupport` has an `@_exported import Foundation`, we can just remove these few lines entirely. The only reason they're spelled out here is because `@_exported` doesn't "re-export" the `@testable`-ness of the import. Same goes for any files that have changes like this

Reply:
Removed all imports to `Foundation(Essentials|Internationalization|)` when `TestSupport` is imported here: a23e682df57dc0fc0f59573e8d1cf0f64d49ea98